### DISCUSSION TITLE
will LTBEEF ever work again?

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#192] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
just asking, since my chromebook hasn't been updated for 2 months and my school is trying to force us to turn them in to be updated.

**Comment by The-Man-Is-Me** [comment, marked as answer]
Yes, These people are working on it and others are as well. Probably give it a month.

**Reply by ghost** [reply]
you can also use the ones on my profile

**Reply by The-Man-Is-Me** [reply]
yep, tons of options
